x/group/spec/messages.md

```diff
@@ -0,0 +1,46 @@
+```go
+// Groups get their own GroupID
+type MsgCreateGroup struct {
+	Signer sdk.AccAddress `json:"signer"`
+	// The Owner of the group is allowed to change the group structure. A group account
+	// can own a group in order for the group to be able to manage its own members
+	Owner  sdk.AccAddress `json:"owner"`
+	// The members of the group and their associated weight
+	Members []Member `json:"members,omitempty"`
+	// TODO maybe make this something more specific to a domain name or a claim on identity? or Info leave it generic
+	Memo string `json:"memo,omitempty"`
+}
+
+// group accounts get their own sdk.AccAddress
+type MsgCreateGroupAccount struct {
+	Signer         sdk.AccAddress `json:"signer"`
+	// The Owner of a group account is allowed to change the DecisionPolicy. This can be left nil 
+	// in order for the group account to "own" itself
+	Owner          sdk.AccAddress `json:"owner"`
+	Group          GroupID        `json:"group"`
+	DecisionPolicy DecisionPolicy `json:"decision_policy"`
```

Comment:
Note, this is an interface, defined below: https://github.com/cosmos-cg-key-management/cosmos-sdk/pull/13/files#diff-370c71de77eed90437dda7009b8ca545R89